src/main/java/com/juniorstart/juniorstart/controller/JobOfferController.java

```diff
@@ -0,0 +1,36 @@
+package com.juniorstart.juniorstart.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import com.juniorstart.juniorstart.payload.JobOfferRequest;
+import com.juniorstart.juniorstart.service.JobOfferService;
+import jakarta.validation.Valid;
+
+@RestController
```

Comment:
Hmm, myślę, że można zacząć to klas kontrolerów dawać @RequestMapping("api\v1\...")
Tutaj myślę, że dobrą ścieżką byłoby api\v1\jobOffer albo job same, nie wiem które lepsze :P

Reply:
Ja bym ustawił @RequestMapping("/api/jobs_offer") dla klasy kontrolera i dla każdej metody był dawał @PostMapping, @PutMapping itp

Reply:
Ważne jest by w linku było te v1. Jak będziemy na przykład poprawiać kod, to nadal będzie działać stare API w razie czego, a nowe będzie pod v2.